src/components/CreateSandboxForm.vue

```diff
@@ -36,8 +48,9 @@ export default {
 				body: JSON.stringify( {} ), // recaptcha goes here?
 			} ) ).json();
 
-			this.signInLink = response.signInLink;
-			this.showSuccessDialog = true;
+			this.signInLink = "https://" + response.data.domain;
```

Comment:
In the unhappy path `response.data` might be undefined so it'll error before we can show the failure dialog. Can fix that later, though :)